### PR TITLE
Add missing registry examples to Ansible Broker Configuration

### LINKED ISSUE #13920: [enterprise-3.11] Edit suggested in file install_config/oab_broker_configuration.adoc
### Which section(s) is the issue in?
Registry Configuration

### What needs fixing?

More documentation is needed around what registry types are supported and examples of their configuration.
Currently there are examples of Mock, Dockerhub, and Ansible Galaxy, Local OpenShift,  RHCC, RHCC Partner, Multiple.
We are missing examples for Helm Chart, API v2, and Quay.

Examples of all types can be found in the ansible-service-broker repository:
https://github.com/openshift/ansible-service-broker/blob/master/docs/config.md#registry-configuration

### PR DESCRIPTION
**Pull request by nccurry**
Fixes #13920

### CODE REVIEW

**Comment by vikram-redhat** [comment]
@bergerhoffer - when you have a chance, PTAL.

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: 1. Can you replace `*Note:* A heavy...` with `NOTE: A heavy...` so that it renders as a note?
2. Can you replace `OpenShift` with `{product-title}`?

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: Can you instead say:

```
See the link:https://github.com/ansibleplaybookbundle/helm2bundle/wiki/Known-Issues[Known Issues] for more information.
````

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: This sentence runs on. Can you reword "can be configured with the apiv2 registry adapter" into a second sentence?

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: This should not have source,yaml here since it's a terminal command. You can just remove this line.

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: This should not have source,yaml here since it's a terminal command. You can just remove this line.

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: Can you put "$" in front of this command? (Or "#" if it needs to be run by root)

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: Can you put "$" in front of this command? (Or "#" if it needs to be run by root)

File: install_config/oab_broker_configuration.adoc
Comment (bergerhoffer) [inline install_config/oab_broker_configuration.adoc]: Can you reword this a bit to something like:

```
If an authentication token is provided, private repositories that the token is configured to access and public repositories in the specified organization will load.
```

**Review by bergerhoffer — CHANGES_REQUESTED** [review]
@nccurry Thanks a lot for submitting these updates. Just a few things if you wouldn't mind updating.

**Comment by bergerhoffer** [comment]
@chengzhang1016 Would you mind reviewing these updates as well? Or let me know if there's someone else who should provide QE review. Thanks!

**Comment by nccurry** [comment]
Thanks for taking the time to look at this!
Requested modifications added.

**Comment by bergerhoffer** [comment]
Thanks for the updates @nccurry! Once we get QE approval, I'll merge and cherry-pick this to enterprise-3.11.

**Review by bergerhoffer — APPROVED** [review]

**Comment by kalexand-rh** [comment]
@chengzhang1016, will you PTAL?

**Comment by chengzhang1016** [comment]
@bergerhoffer @kalexand-rh Looks you are adding these changes in document of enterprise-3.11. We need one day to review and test, and will provide feedback by the end of tomorrow.

I and @zihantang-rh will review it.

**Comment by zihantang-rh** [comment]
LGTM, thanks

**Comment by chengzhang1016** [comment]
The changes LGTM. Thx.

**Comment by bergerhoffer** [comment]
Thanks @chengzhang1016 and @zihantang-rh!

Merging and cherry-picking to enterprise-3.11.

**Pull request merged**
